cdap-ui/app/tracker/templates/main.html

```diff
@@ -18,9 +18,42 @@
   <div class="form-group has-feedback tracker-search" ng-keypress="MainController.search($event)">
     <input type="text"
            class="form-control"
-           placeholder="Search datasets"
+           placeholder="Search"
            ng-model="MainController.searchQuery"
-           pattern="^(?![*]).*$">
+           pattern="^(?![*]).*$"
+           cask-focus="searchField">
     <span class="fa fa-search form-control-feedback"></span>
   </div>
+
+  <div class="search-tips">
+    <h3 class="title">
```

Comment:
Not sure if `h3` is needed. We should start enforcing headings to be specific tags. This will become hard when we want to standardize all the headings across CDAP. 

The very least lets not use h3 tag here as we are using h3 and modifying the font size anyways. A div could be sufficient here with a margin bottom.

Reply:
I would vote against using a div, since Google's a11y guidelines say we should <h1>-<h6> tags over div and span, since that makes it easier to navigate the page using a screen reader. If we need to increase the font size with css, maybe use h2 or h1 instead?

Reply:
The reason I suggested div is because we can't use another heading tag here. I agree when we standardize we should use appropriate heading tags so that we satisfy the a11y requirements